app/assets/stylesheets/urls.scss

```diff
@@ -4,6 +4,23 @@ p {
   margin-top: 70px;
 }
 
-.urls-index-header {
-  padding-top: 80px;
+.index-header{
+  margin-top: 50px;
+  margin-left: 30px;
+}
+
+.search-icons {
+  color: black;
+  font-size: 60px;
+}
+
+.index-well {
+  background: #797979;
+}
+
+.nav-icons {
+  font-size: 15px;
+  a:hover {
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.